### PR TITLE
Support Windows newlines in analyzer wire logs

### PR DESCRIPTION
**Pull request by DanTup**
Otherwise this log is all on one line on Windows which is annoying :)

### CODE REVIEW

**Comment by natebosch** [comment]
Can you check whether https://github.com/natebosch/dart_language_server/pull/44 behaves correctly on windows? I'd prefer to use `writeln` to handle the eol differences if that works.

**Comment by DanTup** [comment]
Sure, I'll test when back on Widows shortly. I couldn't see an obvious way to use `writeln` but that would also be my preference! 👍

File: lib/src/shim.dart [hunk 1]
```diff
@@ -21,10 +22,12 @@ import 'utils/guid.dart';
 import 'utils/package_dir_detection.dart';
 import 'utils/per_file_pool.dart';
 
+final eol = Platform.isWindows ? '\r\n' : '\n';
```
Comment (natebosch) [inline lib/src/shim.dart]: [nit] let's make this private.

Comment (DanTup) [reply lib/src/shim.dart]: Doh; used to private-by-default! Fixed :)

**Review by natebosch — APPROVED** [review]

**Pull request merged**